app/lib/primer/classify.rb

```diff
@@ -252,9 +252,11 @@ def extract_value(memo, key, val, breakpoint)
           memo[:classes] << "#{key.to_s.dasherize}#{breakpoint}-n#{val.abs}"
         elsif key == BOX_SHADOW_KEY
           memo[:classes] << if val == true
-                              "box-shadow"
+                              "color-shadow-small"
+                            elsif val == :none
```

Comment:
Should we check for `val == false` too?

Reply:
hmm we never used it, but don't see why not